client/src/markerCollection.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Marker from './marker';
+
+const MarkerCollection = (props) => {
+  const { parklets, displayModal } = props;
```

Comment:
Destructure in function params

Reply:
Forgot about this trick, thanks for the share!